WordPress/Classes/Utility/ContainerContextFactory.swift

```diff
@@ -27,6 +27,13 @@ class ContainerContextFactory: NSObject, ManagedObjectContextFactory {
                 completionBlock?()
             }
         }
+
+        /// Ensure that the `context`'s concurrency type is not `confinementConcurrencyType`, since it will crash if `perform` or `performAndWait` is called.
+        guard context.concurrencyType == .mainQueueConcurrencyType || context.concurrencyType == .privateQueueConcurrencyType else {
```

Comment:
I can do `if context.concurrencyType == .confinementConcurrencyType {`, but Xcode will throw out a warning since we're typing a deprecated enum 🙃